Source/Classes/Internal/HeightContraintOverlayTranslationController.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-class HeightContrainstOverlayTranslationController: OverlayTranslationController {
+class HeightContraintsOverlayTranslationController: OverlayTranslationController {
```

Comment:
The file's name is wrong. Both should be named HeightConstraintOverlayTranslationController.